ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1408,6 +1408,7 @@ SCHWTC:
   - cooling_percentage_command
   - cooling_request_count
   - cooling_run_command
+  - cooling_run_status
```

Comment:
What is the analytical benefit in bringing in this field? You should be able to use the chiller run statuses to infer if cooling is running.

Reply:
@shambergoldstein thank you for your review on that, cooling_run_status for SCHWTC request will be deleted/closed.

Reply:
please remove it from this PR as well, then I should be good to approve the FCU type